release/serve_tests/app_config.yaml

```diff
@@ -0,0 +1,14 @@
+base_image: "anyscale/ray:1.4.1"
+env_vars: {}
+debian_packages:
+  - curl
+  - unzip
+
+python:
+  pip_packages: []
+  conda_packages: []
+
+post_build_cmds:
+  - pip uninstall -y ray || true
+  - pip3 install -U {{ env["RAY_WHEELS"] | default("ray") }}
+  - 'rm -r wrk || true && git clone https://github.com/wg/wrk.git /tmp/wrk && cd /tmp/wrk && make -j && sudo cp wrk /usr/local/bin'
```

Comment:
move this up in the array so it can be cached each time

Reply:
can you elaborate a bit more how our build cache works in this context ?